content/pages/constitution.md

```diff
@@ -385,7 +385,65 @@ Committee office, M3 1003.
 5. Once the committee has been chosen, the Chair of the WiCS Committee must
    ratify the new committeee.
 
-## iii. Standing Committees ##
+## iii. Resignations and Removal of Members ###
+
+1.  Members may resign from the committee at any time by providing notice to all
+    current committee members.
+
+2.  Any member of the committee may initiate the process for removing a
+    committee member, provided the following:
+
+    1. Committee members may only be removed from the committee if they commit
+       serious violations of policies. These include failure to uphold
+       responsibilities assigned by the Constitution and committee, unauthorized
+       actions on behalf of the committee that lead to negative outcomes, and
+       serious violations of policy and procedure that run contrary to committee
+       values;
+
+    2. The committee member who initiated the removal process must provide
+       sufficient evidence to substantiate the claim that the committee member
+       being considered for removal has committed a serious violation of policy.
+       This evidence shall be provided to the Chair privately, who shall make
+       the determination as to whether the evidence is sufficient to bring the
+       matter to the committee. If the member wants to initiate the removal
+       process against the Chair, they shall bring their case to the Secretary,
+       who will make the determination;
+
+    3. If the Chair or Secretary determines that the claim is valid, they shall
+       call a closed session meeting of the committee to discuss the complaint.
+       Minutes of this meeting must be confidential, and any discussion of the
+       matter must occur privately. In particular, any discussion of the matter
+       shall not occur on mediums that are archived for future use, such as
+       mailing list archives and chat logs;
+
+    4. The member being removed has a right to access all evidence presented
+       against them and make a rebuttal;
+
+    5. All committee members, including the member being removed, shall vote on
+       the removal of the member. The member may only be removed with a
+       supermajority of votes. The outcome of the decision and a brief
+       explanation for the decision shall be recorded in the non-confidential
+       minutes;
+
```

Comment:
A few lines up we referred the member being removed to as "the member being considered for removal" could we put that language here as well since it's more accurate